controllers/hetznerbaremetalhost_controller_test.go

```diff
@@ -17,11 +17,20 @@ limitations under the License.
 package controllers
 
 import (
+	"testing"
 	"time"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	infrav1 "github.com/syself/cluster-api-provider-hetzner/api/v1beta1"
```

Comment:
this will mess with the linting. YOu need to put the caph dependencies below. Maybe make format or something should solve this?

Reply:
I fixed that via `make format-golang`.